src/main/java/com/greghaskins/spectrum/Spectrum.java

```diff
@@ -223,6 +224,30 @@ public static Configuration configure() {
 
 
   /**
+   * Implements a rules context within the current suite using the given rules class as a mix-in.
+   * These rules will cascade down and be applied at the level of specs or atomic specs.
+   * @param rulesClass type of object to create and apply rules to for each spec.
+   * @param <T> type of the object
+   * @return a supplier of the rules object
+   */
+  public static <T> Supplier<T> applyRules(final Class<T> rulesClass) {
```

Comment:
Renamed `junitMixin` - it shall stay here because the `junit` package is presently `internal` and should stay that way. I think `Spectrum` is quite a good Facade, but I appreciate you'll want to restructure and move packages about, so you'll have more freedom if I leave this alone.